cmake/modules/platform/os/zos.cmake

```diff
@@ -20,6 +20,7 @@
 #############################################################################
 
 list(APPEND OMR_PLATFORM_DEFINITIONS
+	-D_AE_BIMODAL
```

Comment:
Should this be contributed by `openxl.cmake` instead? Or will this work with `xlc` as well?

Reply:
This works with `xlc` as well. I checked this with the compiler team and had to make sure it compiles with both using `AE_BIMODAL` , because after removing the redefinitions from `ctype.h` , it broke the `xlc` builds as well. So adding it here was intentional.